### ISSUE TITLE
`NullReferenceException` in `RdioObjectStore.Add()`

### ISSUE THREAD

**Issue by lewurm** [abock/conservatorio#1]
```
$ ./conservatorio\ \(1\) 'bernhard.urban@xamarin.com'
Conservatorio v1.0.1.0 (master/288156a, Mon Nov 23 22:32:57 UTC 2015)
  by Aaron Bockover (@abock)
  http://conservator.io

Starting work for 'bernhard.urban@xamarin.com'...
  * Resolved user Bernhard Urban (s35569418)...
  * Fetching keys...
    11 toplevel keys of interest
  * Fetching objects...
    [#                             ] 0.0% (0 / 11) System.NullReferenceException: Object reference not set to an instance of an object
  at Conservatorio.Rdio.RdioObjectStore.Add (Conservatorio.Rdio.RdioObject o) <0x39cc728 + 0x00010> in <filename unknown>:0
  at Conservatorio.Rdio.Api+<LoadObjectsAsync>c__asyncA.MoveNext () <0x39bdce0 + 0x00197> in <filename unknown>:0
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () <0x39ce070 + 0x0002b> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) <0x39cdd60 + 0x000b3> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) <0x39cdc18 + 0x00083> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) <0x14212f8 + 0x00033> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () <0x21a1518 + 0x00013> in <filename unknown>:0
  at Conservatorio.UserSyncController+<KeyProcessorAsync>c__async2.MoveNext () <0x39bbcb8 + 0x001b7> in <filename unknown>:0
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () <0x39ce070 + 0x0002b> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) <0x39cdd60 + 0x000b3> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) <0x39cdc18 + 0x00083> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) <0x14212f8 + 0x00033> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () <0x21a1518 + 0x00013> in <filename unknown>:0
  at Conservatorio.ItemsProcessor`1+<CoreProcessAsync>c__async1[T].MoveNext () <0x21b55b8 + 0x0046f> in <filename unknown>:0
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () <0x39ce070 + 0x0002b> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) <0x39cdd60 + 0x000b3> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) <0x39cdc18 + 0x00083> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) <0x14212f8 + 0x00033> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () <0x21a1518 + 0x00013> in <filename unknown>:0
  at Conservatorio.ItemsProcessor`1+<ProcessAsync>c__async0[T].MoveNext () <0x21b1128 + 0x002c7> in <filename unknown>:0
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () <0x39ce070 + 0x0002b> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) <0x39cdd60 + 0x000b3> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) <0x39cdc18 + 0x00083> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) <0x14212f8 + 0x00033> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () <0x21a1518 + 0x00013> in <filename unknown>:0
  at Conservatorio.UserSyncController+<SyncStepAsync>c__async0.MoveNext () <0x14206b8 + 0x0080f> in <filename unknown>:0
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () <0x39ce070 + 0x0002b> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) <0x39cdd60 + 0x000b3> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) <0x39cdc18 + 0x00083> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) <0x14212f8 + 0x00033> in <filename unknown>:0
  at System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () <0x14212c8 + 0x00013> in <filename unknown>:0
  at Conservatorio.ConsoleApp+<SyncUser>c__async0.MoveNext () <0x141dcb8 + 0x002fb> in <filename unknown>:0
```

**Comment by abock** [comment]
Confirmed - I'll look into this after dinner. Thanks!

**Comment by lewurm** [comment]
dunno if that could be the source of the problem, but I only have a hidden playlist on my account.

**Comment by abock** [comment]
Fixed by PR #2. Will release update shortly.

**Comment by abock** [comment]
Fixed in 1.2: http://conservator.io

Thanks!
